### ISSUE TITLE
Ignores devices if they have same name

### ISSUE THREAD

**Issue by xpatohr** [icanos/hassio-plejd#91]
Thanks for a great plugin!

I have several Plejd devices in my home (+30).
Many of them have the same name, but in different rooms.

For example 'Living Room' has a lamp called 'Ceiling lamp' and it will be imported, but the lamp in 'Bedroom 1' called 'Ceiling lamp' will be ignored during import because 'Ceiling lamp' already exists from 'Living Room'

```
Entity id already exists - ignoring: light.taklampa. Platform mqtt does not generate unique IDs
10:03:12 PM – Light (ERROR) - message first occurred at 10:03:11 PM and shows up 25 times
```
```
Log Details (ERROR)
   Logger: homeassistant.components.light
   Source: helpers/entity_platform.py:439
   Integration: Light (documentation, issues)
   First occurred: 10:03:11 PM (25 occurrences)
   Last logged: 10:03:12 PM

Entity id already exists - ignoring: light.taklampa. Platform mqtt does not generate unique IDs
Entity id already exists - ignoring: light.fonstersmyg. Platform mqtt does not generate unique IDs
Entity id already exists - ignoring: light.spotlight. Platform mqtt does not generate unique IDs
Entity id already exists - ignoring: light.led_list. Platform mqtt does not generate unique IDs
Entity id already exists - ignoring: light.spegel. Platform mqtt does not generate unique IDs
```

I have tried both 'includeRoomsAsLights set to true' and 'includeRoomsAsLights set to false' with the same result.

Is there any way to get around this? Or possibility to include the room name from Plejd in the entities id?
For example, default id 'light.room.name' instead of 'light.name'

_Thanks_

**Comment by Lirilesh** [comment]
I have the exact same issue. I had to rename them for them to work, but as you say, I have ceiling lamps i many rooms and wish to keep them that way for it to make sense..

**Comment by xpatohr** [comment]
> I have the exact same issue. I had to rename them for them to work, but as you say, I have ceiling lamps i many rooms and wish to keep them that way for it to make sense..

Yeh, same here. I like to avoid to call 'ceiling light' different name all rooms with a 'celling light'. Hard to understand the name standard otherwise :-)

**Comment by icanos** [comment]
I'll fix this, haven't thought about it :-D

**Comment by xpatohr** [comment]
> I'll fix this, haven't thought about it :-D

Awesome! Thanks Markus.

**Comment by xpatohr** [comment]
[Created pull request 102.](https://github.com/icanos/hassio-plejd/pull/102)

**Comment by SweVictor** [comment]
Just a not that #161 also is a mqtt command clash but due to a different reason. Not necessarily in the scope of this PR, but might be.

**Comment by SweVictor** [comment]
Resolved in #179, merged to develop branch
